package.json

```diff
@@ -1,7 +1,7 @@
 {
-    "name": "web3",
+    "name": "web3.js",
```

Comment:
What is the reason to rename the root/dev package from web3 to web3.js? Also this is a non-required change :-)

Reply:
We were running into lerna conflicts/bugs in our release process because the root package name was identical to one of the subpackages, so this fixed that and simplified our latest [release steps](https://github.com/ethereum/web3.js/blob/81e6cd5bf25cb13c7081e8c770eebc2416726965/RELEASE.md#release-candidate-rc-release-procedure).

Reply:
hmm.. never had that.. 🤔 (typically lerna).. it should ignore this root ``package.json`` at all especially because of the ``private: true``

Reply:
Probably okay, i.e. it's not uncommon to have the `"name"` be `"root"` in the root `package.json` file.

Reply:
See e.g. https://github.com/embarklabs/embark/blob/master/package.json#L41

Reply:
Yep, just commented it to have the reason documented. :-) @michaelsbradleyjr thanks!